preferences.yaml

```diff
@@ -0,0 +1,76 @@
+swagger: '2.0'
+info:
+  version: 0.0.1
+  title: Preferences API
+  description: >-
+    The API lets you manage attributes associated with users (user preferences).
+
+    You can get any attribute for any user (read only) according to the rights
+    you owe on the user resource. 
+
+    There rights are set up at moment you've subscribed the service exposed by
+    this API.
+  license:
+    name: MIT
+host: localhost
+basePath: /api/v1
+schemes:
+  - https
+paths:
+  '/users/{fiscal_number}/preferences':
```

Comment:
GET /users/{fiscal_number}/preferences/{preference_id} would be useful in the case of resource versioning. We can now assume that "/users/{fiscal_number}/preferences" always points to the latest revision. Moreover a user it's not his preferences so the '/User' endpoint should be free IMHO. Said that, it's ok for me to use "/preferences/{fiscal_number}". Should we settle on it?